src/components/Button/Button.tsx

```diff
@@ -105,7 +105,7 @@ class Button extends UIComponent<any, any> {
               key="btn-icon"
               name={icon}
               xSpacing={!content ? 'none' : iconIsAfterButton ? 'before' : 'after'}
-              color={primary ? 'white' : 'black'}
+              variables={{ color: primary ? 'white' : 'black' }}
```

Comment:
actually, this still will result in Icon having color different from the button's one (obviously, Icon could be either black or white) - while, as a consumer, I would rather expect that it would take foreground color of Button